lib/commands/touch.js

```diff
@@ -1,9 +1,51 @@
 const commands = {};
-
 //This is needed to make clicks on -image elements work properly
 commands.performActions = async function (actions) {
-  return await this.winAppDriver.sendCommand('/actions', 'POST', {actions});
+  switch (actions[0].id) {
```

Comment:
why only the first action gets processed, - there might be many?

Reply:
The format of the json is an array for a single object containing the id and a sub array with all the actions themselves. The initial structure is always the same, the subarray of actions can contain multiple actions
{
   "actions":[
      {
         "type":"key",
         "actions":[
            {
               "type":"keyDown",
               "value":""
            },
            {
               "type":"keyUp",
               "value":""
            }
         ],
         "id":"default keyboard"
      }
   ]
}

Reply:
I'm not sure this is the case. According to https://www.w3.org/TR/webdriver/#actions multiple ids are possible. Each of these ids may, for example, label a different finger for a multitouch actions or mouse and keyboard gestures executes simultaneously

Reply:
You are right, I totally missed the fact that actions from different inputs can be chained. I'll modify the structure and order of the operations